pages/implementers/interfaces.md

```diff
@@ -121,6 +121,14 @@ In some sense, two argument validation is the simplest possible API for JSON Sch
 An API which attempts to prepare a schema for repeated use.
 It may (and likely will) perform some form of preoptimization, performing some set of work ahead of time such that it will not be necessary to repeat when validating many instances.
 
+### Subschema Validation
+
+**Type**: `Schema` × `String` × `Instance` → `Result`
+
+An API which supports validating an instance against a *subschema* contained within the given schema.
+
+The subschema is typically identified via a JSON Path, or equivalent syntax, and is as opposed to validation using a new schema referencing the subschema via the `$ref` keyword.
```

Comment:
I would prefer mentioning JSON pointer instead of JSON Path.  Pointers identify single locations.  JSON Path is a query syntax.

I realize that the inspiring library probably use JSON Path, but I think it does so erroneously.

Reply:
That was actually a typo and I indeed meant to mention JSON Pointer :) Thanks!